src/index.js

```diff
@@ -12,6 +12,12 @@ const IOS_MODAL_HEIGHT = 262;
 
 export default class RNPickerSelect extends PureComponent {
     static propTypes = {
+        // A platform-dependent reference to an element that exposes a best-effort `focus` method that doesn't involve
+        // opening the picker
+        focusRef: PropTypes.oneOfType([
```

Comment:
I picked a custom name for the ref, as forwarding the real ref broke literally [the whole testsuite](https://github.com/Expensify/react-native-picker-select/blob/84ee97dec11c2e65609511eb5a757d61bbeeab79/test/test.js#L98). It's all built on the assumption that the `ref` actually points to the actual `RNPickerSelect` component.